docroot/themes/humsci/humsci_basic/src/scss/tools/_functions.general.scss

```diff
@@ -0,0 +1,8 @@
+// Calculate rems based upon the root font size
+// of 10px which matches the root font size in
+// Decanter ($su-root-font-size)
+// set in /humsci_basic/src/scss/settings/_variables.general.scss
+@function calculate-rems($size-in-pixels) {
```

Comment:
I really hate to do this but all of our mixins and functions need to be prefixed. This should be `hb-calculate-rems()`.

Reply:
Oh yes! 🙈 No big deal. Find & replace is my friend. :smile: